contexts/mooncats/index.tsx

```diff
@@ -80,7 +80,7 @@ export const MooncatsProvider: React.FC = ({ children }) => {
   const getContract = () => {
     if (!signer) return undefined;
     return new ethers.Contract(
-      "0x60cd862c9C687A9dE49aecdC3A99b74A4fc54aB6",
+      '0x60cd862c9C687A9dE49aecdC3A99b74A4fc54aB6',
```

Comment:
noticed we are using single quotes everywhere, do you want to enforce this style with husky? so that the next time someone works on our repo, they do not revert to double quotes and change half the repo lol